src/char/char.h

```diff
@@ -29,6 +29,19 @@ enum {
 	TABLE_GUILD_STORAGE,
 };
 
+typedef enum e_makechar_error {
+	ERROR_MK_SQL = -255,
```

Comment:
`MAKECHAR_MAKE_SQL`. Is "MK" supposed to be short for "MAKE"? Best to type it all out.